scripts/build/binary

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
👍 This was causing builds to fail in [ubuntu-xenial](https://askubuntu.com/a/504549) containers due to the default `/usr/bin/env sh` not having `source`